apps/demos/Demos/Form/SmartPaste/Angular/app/app.component.ts

```diff
@@ -0,0 +1,189 @@
+import { NgModule, Component, ViewChild, enableProdMode } from '@angular/core';
+import { BrowserModule } from '@angular/platform-browser';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { AzureOpenAI, type OpenAI } from 'openai';
+
+import { DxButtonModule, DxTextAreaModule } from 'devextreme-angular';
+import type { ButtonStyle } from 'devextreme-angular/common';
+import {
+  AIIntegration,
+  RequestParams,
+  Response,
+} from 'devextreme-angular/common/ai-integration';
+import { DxFormModule, DxFormComponent } from 'devextreme-angular/ui/form';
+import notify from 'devextreme/ui/notify';
+
+import { Service } from './app.service';
+
+if (!/localhost/.test(document.location.host)) {
+  enableProdMode();
+}
+
+let modulePrefix = '';
+// @ts-ignore
+if (window && window.config?.packageConfigPaths) {
+  modulePrefix = '/app';
+}
+
+const stylingMode = 'filled';
+
+type AIMessage = (OpenAI.ChatCompletionUserMessageParam | OpenAI.ChatCompletionSystemMessageParam) & {
+  content: string;
+};
+
+const showNotification = (message: string, of: string, isError?: boolean, offset?: string) => {
+  notify({
+    message,
+    position: {
+      my: "bottom center",
+      at: "bottom center",
+      of,
+      offset: offset ?? '0 -50',
+    },
+    width: 'fit-content',
+    maxWidth: 'fit-content',
+    minWidth: 'fit-content',
+  }, isError ? 'error' : 'info', 1500);
+}
+
+@Component({
+  selector: 'demo-app',
+  templateUrl: `.${modulePrefix}/app.component.html`,
+  styleUrls: [`.${modulePrefix}/app.component.css`],
+  providers: [Service],
+})
+export class AppComponent {
+  @ViewChild(DxFormComponent, { static: false }) form: DxFormComponent;
+
+  colCountByScreen = {
+    xs: 2,
+    sm: 2,
+    md: 2,
+    lg: 2,
+  };
+
+  amountDueEditorOptions = { placeholder: '$0.00', stylingMode };
+  amountDueAIOptions = { instruction: 'Format as the following: $0.00' };
+
+  statementDueEditorOptions = { placeholder: 'MM/DD/YYYY', stylingMode };
+  statementDueAIOptions = { instruction: 'Format as the following: MM/DD/YYYY' };
+
+  textEditorOptions = { stylingMode };
+
+  phoneEditorOptions = { placeholder: '(000) 000-0000', stylingMode };
+  phoneAIOptions = { instruction: 'Format as the following: (000) 000-0000' };
+
+  emailAIOptions = { instruction: 'Do not fill this field if the text contains an invalid email address. A valid email is in the following format: email@example.com' };
+
+  zipEditorOptions = { stylingMode, mode: 'text', value: null };
+  zipAIOptions = { instruction: 'If the text does not contain a ZIP, determine the ZIP code from the provided address.' };
+
+  resetButtonOptions = {
+    stylingMode: 'outlined' as ButtonStyle,
+    type: 'normal'
+  };
+
+  smartPasteButtonOptions = {
```

Comment:
```suggestion
  smartPasteButtonOptions: DxButtonTypes.Properties = {
    stylingMode: 'contained',
    type: 'default',
  }
```
same for other frameworks